src/index.js

```diff
@@ -0,0 +1,91 @@
+// import _ from "lodash";
+// import printMe from "./print.js";
+import './style.css';
+
+// function component() {
+//   const element = document.createElement("div");
+//   const btn = document.createElement("button");
+
+//   element.innerHTML = _.join(["Hello", "webpack"], " ");
+//   element.classList.add("hello");
+
+//   btn.innerHTML = "Click me and check the console!";
+//   btn.onclick = printMe;
+
+//   element.appendChild(btn);
+
+//   return element;
+// }
+
+// document.body.appendChild(component());
+
+/* eslint-disable no-restricted-syntax */
```

Comment:
- [ ]  Please avoid disabling linter rules, the linters help tremendously to keep a consistent code structure and enforce some best practices over your code! Kindly consider removing this comment and fixing the error. 🤓 🔧